ecosystem/typescript/sdk/package.json

```diff
@@ -54,5 +54,5 @@
     "tweetnacl": "^1.0.3",
     "yarn": "^1.22.18"
   },
-  "version": "1.0.0"
+  "version": "1.1.0"
```

Comment:
Is this a bump because if I was using the types they no longer import the same?

Reply:
You can still import the same way. This PR should be backwards compatible. However, developers are able to `import {TxnBuilderTypes, BCS} from aptos` going forward. 

To bump the version is mainly because we added more functionalities, such as multisig, etc. Per SemVer, https://semver.org/, the minor version should be bumped.